worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -499,6 +508,15 @@ class PushDataHandler extends BaseMessageHandler with Logging {
     }
     fileWriters.foreach(_.incrementPendingWrites())
 
+    val closedFileWriter = fileWriters.find(_.isClosed)
```

Comment:
sorry for the later review, I think we should use the `while` loop, like what we did in https://github.com/apache/incubator-celeborn/pull/1668